styles.css

```diff
@@ -46,3 +46,14 @@ h1 {
 #bee {
   margin-left: 160px;
 }
+#flexbox {
```

Comment:
Please try not to style with `id`s. The reason is that an `id` is unique for the whole document and you can only use it once. It is always preferable to use a class.